tests/test_controller.py

```diff
@@ -299,7 +301,7 @@ def test_library_details(self):
         eq_(uuid, response.get("uuid"))
 
         self._check_keys(response)
-        self._is_library(library, response)
+        self._is_library(self.nypl, response)
```

Comment:
I think the old code is better -- this will cause `nypl` to be created again, when it doesn't look like that is necessary.

Reply:
The old code suddenly started failing--don't know why...  It's still instantiating `library` correctly, but then by the time it's ready to call `_is_library`, `library.__dict__` suddenly contains a sqlalchemy InstanceState object and nothing else.